### DISCUSSION TITLE
Git Hub Education Approved not Available

### DISCUSSION THREAD

**Discussion by razinrayees** [community/community#129836] [GitHub Education] [answered]
### Hi everyone,

Git Hub Education Approved not Available

**Comment by davevad93** [comment]
Hi @razinrayees ,

What's the status of your application? Benefits are available when it says "`Congrats! Your request for @razinrayees was approved and your benefits are now available`" if it states "`Your benefits will be available soon`" it means that you still have to wait a few days.

**Reply by razinrayees** [reply]
It's showing your benefits will be available soon

**Reply by davevad93** [reply]
Don't worry then, just wait a few more days, sometimes it may take a little longer depending on different factors such as the volume of requests. When the benefits are available the `Approved` status label will turn purple.

**Reply by razinrayees** [reply]
Its been more than four days

**Reply by davevad93** [reply, marked as answer]
Hey @razinrayees , see this discussion #111352 which states the following:

- ***We aim to provide you with your educational benefits within 3 days of this approval status. However, during busy periods like the start of a new semester, it may take a bit longer.***

  ***If you have not received your education benefits within 5 days of your approval status please raise a discussion on the [GitHub Education community](https://github.com/orgs/community/discussions/categories/github-education) and the Education community team will be able to assist in resolving this for you. 🌟***

Keep in mind that it could take even longer, I have seen cases where it has taken a couple of weeks, so be patient, if in a few days you still don't have your benefits, someone from the GitHub Education Staff will assist you.

**Comment by Ak4shKr** [comment]
It has been 3 days to verified id but still student benefit pending. what should I do?

**Reply by changabaliye** [reply]
did you get your educational benefit
